### PR TITLE
add option to ignore linting and types error during deploy

### PR DESCRIPTION
**Pull request by technophile-04**
#### Description
Had a discussion with @carletex about it... sometimes beginners might have a hard time figuring out the correct types / linting errors and it would be handy if we have an option that lets you yolo app quickly

By default its `false` but if someone want to share a quick demo/poc deployed `url` it will be nice. 

#### Approach take : 
Added a flag to `.env.production` which by default is `false` and to ignore errors you run `yarn vercel:yolo` command which sets this flag to `true` or manually set  the flag to `true` and run `yarn vercel`. 

#### Demo : 
Screenshot 2023-02-21 at 3 33 21 PM

^ Here is the SS, vercel skipping both types and linting

### CODE REVIEW

**Comment by carletex** [comment]
We've created a monster. YOLO!! haha

Uncle bob, don't judge us.

We don't need to forget that we are building a tool that (if everything goes well) a lot of people will use to onboard on Ethereum, and some of them won't have TS knowledge, so we want to make their life easier at the beginning, and not let them get frustrated. It's about to find Balance!

This is related to #141 (fixes some of the issues).

File: packages/nextjs/next.config.js [hunk 1]
```diff
@@ -3,6 +3,12 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  typescript: {
+    ignoreBuildErrors: process.env.NEXT_PUBLIC_IGNORE_BUILD_ERROR === "true" ? true : false,
```
Comment (rin-st) [inline packages/nextjs/next.config.js:7]: you can do it without ternary :) `ignoreBuildErrors: process.env.NEXT_PUBLIC_IGNORE_BUILD_ERROR === "true"`

File: packages/nextjs/next.config.js [hunk 2]
```diff
@@ -3,6 +3,12 @@
...
+  },
+  eslint: {
+    ignoreDuringBuilds: process.env.NEXT_PUBLIC_IGNORE_BUILD_ERROR === "true" ? true : false,
```
Comment (rin-st) [inline packages/nextjs/next.config.js:10]: and here

File: packages/nextjs/next.config.js (hunk 1 above)
Comment (technophile-04) [reply packages/nextjs/next.config.js:7]: Trust me I actually spent 4-5 min trying to figure out the best way to write this and this didn't clicked 😂😂, Thanks @rin-st looks better fixed at #197

**Pull request merged**